src/main/java/core/basesyntax/FileWrite.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.io.File;
+
+public interface FileWrite {
+    default void writeFile(File file, String data) {
+
+    }
```

Comment:
```suggestion
    void writeToFile(String toFileName, String data);
```
you should create a File object inside of this method